tests/crash/stack_overflow.c

```diff
@@ -1,7 +1,6 @@
 char toto()
 {
-    char buffer[4096];
-    buffer[0] = 0;
+    char buffer[4096] = {0};
```

Comment:
To test the Python faulthandler module of the Python stdlib, I wrote:

```
    volatile unsigned char buffer[4096];
    buffer[0] = 1;
    buffer[4095] = 0;
```

https://github.com/python/cpython/blob/fcbe0cb04d35189401c0c880ebfb4311e952d776/Modules/faulthandler.c#L1132-L1147

volative prevents compilation optimizations:

```
    /* Allocate (at least) 4096 bytes on the stack at each call.
       bpo-23654, bpo-38965: use volatile keyword to prevent tail call
       optimization. */
```

Reply:
OK, I have changed this.